src/transactions/cdp.js

```diff
@@ -384,17 +384,17 @@ export async function openCDP(openingALGOs, openingGARD, commit, toWallet) {
     let stxn8 = algosdk.signLogicSigTransactionObject(txn8, lsig);
     stxns.push(stxn8.blob)
   }
-  
+
   setLoadingStage("Confirming Transactions...");
-  
+
   let response = await sendTxn(
     stxns,
     "Successfully opened a new CDP.",
   );
-  
+
   addCDPToFireStore(accountID, -openingMicroALGOs, microOpeningGard, 0);
   let completedMint = JSON.parse(localStorage.getItem("gleamMintComplete"))
-  if (!completedMint.includes(info.address)) {
+  if (completedMint != null && !completedMint.includes(info.address)) {
```

Comment:
I think this isn't the intended functionality, and will just make it so the code never executes...Could you revert the changes to the "completedMint" variable in cdp.js? We can have Minh fix it since I believe he wrote this section.